tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/PlacementServiceImpl.java

```diff
@@ -1010,10 +1010,11 @@ public boolean isEligibleForCurrentTraineeWteChangeNotification(final Placement
     LocalDate currentDateFrom = latestApprovedLog.getDateFrom();
 
     return
-        ((updatedPlacementDetails.getWholeTimeEquivalent() != null && updatedPlacementDetails
-            .getWholeTimeEquivalent()
-            .compareTo(currentPlacement.getPlacementWholeTimeEquivalent()) != 0)) &&
-            ((currentDateFrom != null && currentDateFrom
+        (updatedPlacementDetails.getWholeTimeEquivalent() != null && (
+            currentPlacement.getPlacementWholeTimeEquivalent() == null
```

Comment:
This condition of checking null value is newly added, and the rest modified lines are for formating.